RELEASE-NOTES.md

```diff
@@ -1,5 +1,25 @@
 # Prefect Release Notes
 
+## 2.0.4
+
+### Simplified deployments
+The deployemnts experience has been refined to remove extraneous artifacts and make configuration even easier. In particular:
```

Comment:
```suggestion
The deployment experience has been refined to remove extraneous artifacts and make configuration even easier. In particular:
```

Reply:
I suspect "deployments" (plural) might be our preferred way of talking about this feature, but I don't know. I'm going to commit this because it fixes the typo, and we can change it later if we want to.